mobile/lib/features/stable/application/stabilize_service.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:get_10101/ffi.dart' as rust;
+import 'package:get_10101/features/trade/domain/contract_symbol.dart';
+import 'package:get_10101/features/trade/domain/direction.dart';
+
+class OrderService {
```

Comment:
The file here is called StabilizeService but this class is OrderService?

Reply:
This service isn't even used 🙈 Thanks for spotting that :)